helm_deploy/hmpps-prisoner-from-nomis-migration/values.yaml

```diff
@@ -128,6 +128,10 @@ generic-service:
       HMPPS_SQS_QUEUES_MIGRATIONSENTENCING_QUEUE_NAME: "sqs_name"
     sqs-migration-sentencing-dlq:
       HMPPS_SQS_QUEUES_MIGRATIONSENTENCING_DLQ_NAME: "sqs_name"
+    sqs-migration-visitorders-queue:
+      HMPPS_SQS_QUEUES_MIGRATIONVISIT_ORDERS_QUEUE_NAME: "sqs_name"
+    sqs-migration-visitorders-dlq:
+      HMPPS_SQS_QUEUES_MIGRATIONVISITORDERS_DLQ_NAME: "sqs_name"
```

Comment:
just checking that you don't need the underscore on the second one

Reply:
good spot - it's actually the first one I don't want an underscore on - will fix